client/src/components/ScoutContent.js

```diff
@@ -1,64 +1,56 @@
-import React, {Component} from 'react'
-import Form from 'react-bootstrap/Form';
-import Button from 'react-bootstrap/Button';
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 
 class ScoutContent extends Component {
-
```

Comment:
I'm not sure why these are showing as diffs. Maybe just some newline issues. Not a big deal, makes it hard to see diffs, but let's not worry about it.

Reply:
looks like spacing was adjusted. Seems fine. We should create a linter file for whether to use double or single quotations. This way your compiler(webpack)/linter will complain when using them the wrong way.